.github/workflows/on_build_do_test.yml

```diff
@@ -75,7 +84,7 @@ jobs:
         run: >
             cd test_dir/infinispan &&
             ./mvnw verify -V -B -e -DrerunFailingTestsCount=2
-            -Dmaven.test.failure.ignore=true -Dansi.strip=true -Pnative
+            -Dmaven.test.failure.ignore=true -Dansi.strip=true -Pnative -pl
```

Comment:
Why do we need the empty `-pl` arg?

Reply:
no need actually, I thought I pushed the correction. but something went wrong
thanks!